File: Back/src/Cinema.Domain/Sessao.cs
using System;
using Cinema.Domain.Enum;

namespace Cinema.Domain
{
    public class Sessao
    {
        public int Id { get; set; }
        public DateTime DataSessao { get; set; }
        public DateTime HorarioInicial { get; set; }
        public DateTime HorarioFinal { get; set; }
        public decimal ValorIngresso { get; set; }
        public TipoAnimacao TipoAnimacao { get; set; }
        public TipoAudio TipoAudio { get; set; }
        public int FilmeId { get; set; }
        public Filme filme { get; set; }
        public int SalaId { get; set; }
        public Sala sala { get; set; }
    }
}

File: Front/Cinema-App/src/app/models/Sessao.ts
import { TipoAnimacao } from "../Enums/TipoAnimacao.enum";
import { TipoAudio } from "../Enums/TipoAudio.enum";
import { Filme } from "./Filme";
import { Sala } from "./Sala";

export interface Sessao {
  id :  number;
  dataSessao :  Date;
  horarioInicial :  Date;
  horarioFinal :  Date;
  valorIngresso :  number;
  tipoAnimacao :  TipoAnimacao;
  tipoAudio :  TipoAudio;
  filmeId :  number;
  filme :  Filme;
  salaId :  number;
  sala :  Sala;
}
